pkg/minikube/machine/cluster_test.go

```diff
@@ -60,11 +60,17 @@ func RegisterMockDriver(t *testing.T) {
 	}
 }
 
+var defaultNodeConfig = config.Node{
+	Name: viper.GetString("profile"),
```

Comment:
Do you mind inlining this? I don't find the extra level of indirection to be useful for test readability.  I prefer test inputs be absolutely explicit, and this particular  config is small. 

`config.Node{Name: "minikube"}`

I would prefer it if the tests did not reach into global variables like viper flags unless absolutely necessary.